tests/admin_views/tests.py

```diff
@@ -131,6 +131,7 @@ def setUpTestData(cls):
         cls.chap4 = Chapter.objects.create(title='Chapter 2', content='[ insert contents here ]', book=cls.b2)
         cls.cx1 = ChapterXtra1.objects.create(chap=cls.chap1, xtra='ChapterXtra1 1')
         cls.cx2 = ChapterXtra1.objects.create(chap=cls.chap3, xtra='ChapterXtra1 2')
+        Actor.objects.create(name="Palin", age=27)
```

Comment:
Please use single quotes.

Reply:
@felixxm Are you picking these up by eye or linting the diff programatically? If the latter, can I ask your incantation? `flake8` comes out clean with the project config. Ta!

Reply:
@carltongibson By eye, there is no black magic behind 😃 .

Reply:
@felixxm OK. Thanks. I will just continue to sharpen the nit-picker then 😃

Reply:
Wow. Thank you very much guys. I've learned a few things from this PR discussion. Just pushed fixes for these issues.

Also, where we go we don't need no stinkin' automated linters 😉